### ISSUE TITLE
My Ether is gone when i transfered from MEW to Ether delta 

### ISSUE THREAD

**Issue by Morrizen** [etherdelta/etherdelta.github.io#327]
I transfered some ether from MEW to ether delta and the money is missing 
here is the TX ID 

0x101ec4d1dfcc1c17f2d20efbfe5d9be45f67277fbe3fe70975cc9edf40b07105

Some one please help

**Comment by zackcoburn** [comment]
Please ask one of the reps in the chat on the site.

**Comment by Morrizen** [comment]
The chat does not work for me. When i open it it says 404 this is not the chat you are looking for... I lost about 500 dollars i need help please

**Comment by Morrizen** [comment]
This is ridiculous where can i get help... Seriously

**Comment by Morrizen** [comment]
I just lost over 500 dollars

**Comment by zackcoburn** [comment]
https://gitter.im/etherdelta/etherdelta.github.io

On Dec 13, 2017 12:11 PM, "Morrizen" <notifications@github.com> wrote:

I just lost over 500 dollars

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/etherdelta/etherdelta.github.io/issues/327#issuecomment-351284605>,
or mute the thread

**Comment by zackcoburn** [comment]
You used the 0.8 Ether to buy BQX:
https://deltabalances.github.io/tx.html#0xbcb1cc08ba86b0f380d0594a9ac34d7cba5f6b22408cc21b9edaf01180444426

**Comment by Morrizen** [comment]
Yeah apparently there was a buy order that went through when i sent the money... Thanks for the help !
